servlet.py

```diff
@@ -121,6 +124,49 @@ class BaseHandler(tornado.web.RequestHandler):
     def initialize(self):
         self.callback = self.get_argument('callback', default=None)
 
+    @classmethod
+    def initPairsGraph(cls):
```

Comment:
http://stackoverflow.com/questions/4613000/what-is-the-cls-variable-used-in-python-classes Nice, learned something today!